ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6618,48 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+RSM:
+  guid: "e6aa94d5-1860-4a5d-8269-0edb968895e1"
+  description: "Run status monitoring"
+  is_abstract: true
+  uses:
+  - run_status
+  opt_uses:
+  - failed_alarm
+  implements:
+  - OPERATIONAL
+
+SEFC:
```

Comment:
I dont see `supply_air_flowrate_sensor` or `exhaust_air_flowrate_sensor`. Those fields need to be added if this device controls the fan speeds to maintain flowrates to a setpoint.

Secondly, this type could be avoided if you instead apply the types `SFVSC`, `EFVSC`, `SFC`, and `EFC`. Please consider using those abstract types in your canonical type definition.